examples/tutorial/ArithmeticClient.hs

```diff
@@ -24,10 +24,11 @@ main = withGRPCClient clientConfig $ \client -> do
   -- Request for the RunningSum RPC
   ClientWriterResponse reply _streamMeta1 _streamMeta2 streamStatus streamDtls
     <- arithmeticRunningSum $ ClientWriterRequest 1 [] $ \send -> do
-        _ <- send (OneInt 1)
-        _ <- send (OneInt 2)
-        _ <- send (OneInt 3)
-        return ()
+        eithers <- sequence [send (OneInt 1), send (OneInt 2), send (OneInt 3)]
```

Comment:
Err maybe I shouldn't say readable because what you have is perfectly readable, though I think `mapM` or `forM` is more common.